static/app/utils/metrics/fields.tsx

```diff
@@ -59,3 +65,90 @@ export const METRIC_TO_COLUMN_TYPE: Readonly<
   [TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_STALL_LONGEST_TIME]: 'duration',
   [TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_STALL_PERCENTAGE]: 'percentage',
 };
+
+// This is not the full map, just what we need for performance table.
+// In the future we might want something more generic that strips the aggregate functions.
+const DISCOVER_FIELD_TO_METRIC = {
+  'count_unique(user)': `count_unique(${TransactionMetric.SENTRY_TRANSACTIONS_USER})`,
+  'tpm()': `count(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p75(measurements.fcp)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_FCP})`,
+  'p75(measurements.lcp)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_LCP})`,
+  'p75(measurements.fid)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_FID})`,
+  'p75(measurements.cls)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_CLS})`,
+  'p50()': `p50(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p95()': `p95(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p50(transaction.duration)': `p50(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p75(transaction.duration)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p95(transaction.duration)': `p95(${TransactionMetric.SENTRY_TRANSACTIONS_TRANSACTION_DURATION})`,
+  'p75(measurements.app_start_cold)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_APP_START_COLD})`,
+  'p75(measurements.app_start_warm)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_APP_START_WARM})`,
+  'p75(measurements.frames_slow_rate)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_FRAMES_SLOW_RATE})`,
+  'p75(measurements.frames_frozen_rate)': `p75(${TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_FRAMES_FROZEN_RATE})`,
+
+  // TODO(metrics):
+  // transaction.op
+  // http.method
+  // failure_rate()
+  // apdex()
+  // count_miserable(user)
+  // user_misery()
+  // team_key_transaction
+};
+
+const DISCOVER_FIELD_TO_METRIC_GROUP_BY = {
+  transaction: 'transaction',
+
+  // TODO(metrics):
+  // project
+};
+
+const METRIC_TO_DISCOVER_FIELD = invert(DISCOVER_FIELD_TO_METRIC);
+const METRIC_GROUP_BY_TO_DISCOVER_FIELD = invert(DISCOVER_FIELD_TO_METRIC_GROUP_BY);
+
+export function convertDiscoverFieldsToMetrics<T extends string | string[]>(
```

Comment:
Bunch of helpers to convert metric fields to discover fields and the other way around.
There's a high probability we'll be able to get rid of these once the logic is moved over to the API.